tools/sgcollect_info

```diff
@@ -262,6 +259,23 @@ def make_collect_logs_tasks(zip_dir, sg_url):
             task = add_file_task(sourcefile_path=sg_log_file_path)
             sg_tasks.append(task)
 
+
+    # Find log file path from new style logging config
+    guessed_logging_path = extract_element_from_default_logging_config('LogFilePath', config_str)
+    if guessed_logging_path:
+        # Get the parent directory and the log file name
+        log_file_parent_dir = os.path.abspath(os.path.join(guessed_logging_path, os.pardir))
+        log_file_name = os.path.basename(guessed_logging_path)
+        log_file_name_split = os.path.splitext(log_file_name)
+
+        # iterate over all log files, including those with a timestamp prefix
+        rotated_logs_pattern = log_file_parent_dir+os.path.sep+log_file_name_split[0]+"*"+log_file_name_split[1]
+        for log_file_item_name in glob.iglob(rotated_logs_pattern):
```

Comment:
I wasn't sure if wildcard would be treated as a valid path, but I have changed it and tested OK.